charter.md

```diff
@@ -23,48 +23,33 @@ The WebTransport working group will define new client-server protocols
 or protocol extensions in order to support the development of the
 W3C WebTransport API <https://wicg.github.io/web-transport>.
 
-These protocols will support:
-  * Reliable bidirectional and unidirectional communication
-    that provides greater efficiency than Websockets
-    (e.g. removal of head-of-line blocking).
-  * Unreliable datagram communication, functionality not available
-    in Websockets.
-  * Origin checks to allow supporting the Web's origin-based
-    security model.
-
-The WebTransport working group will define three variants:
-  * A protocol directly running over QUIC with its own ALPN.
-  * A protocol that runs multiplexed with HTTP/3.
-  * Fallback protocols that can be used when QUIC or UDP 
-    are not available.
+The WebTransport working group will define a protocol or suite of
+protocols that support a range of simple communication methods.
+These must include unreliable messages (that might be limited by
+the path MTU), reliable messages, and ordered streams of reliable
+messages.  Attention will be paid to the performance of the protocol,
+in particular protocol overheads and the potential for head-of-line
+blocking; its ability to be deployed and used reliably under different
+network conditions; and the ability to integrate protocol use into
+the Web security model.
```

Comment:
I feel like we should probably make the last one mandatory (i.e. anything that can't be used on the Web is out of scope for webtrans wg).